packages/client/src/views/RecordAudit/RecordAudit.tsx

```diff
@@ -363,11 +357,13 @@ function RecordAuditBody({
   }
 
   if (
+    hasScope('record.archive') &&
     isDownloaded &&
     declaration.status &&
-    ARCHIVABLE_STATUSES.includes(declaration.status) &&
-    (userHasRegisterScope ||
-      (userHasValidateScope && declaration.status !== VALIDATED))
+    ARCHIVABLE_STATUSES.includes(declaration.status)
+    // @TODO: Previously here a user with validate scope couldn't archive unless it's validated
+    // (userHasRegisterScope ||
+    // (userHasValidateScope && declaration.status !== VALIDATED))
```

Comment:
In record audit, the record can be archived if `userHasRegisterScope` OR `userHasValidateScope && declaration.status !== VALIDATED)`. 

I replaced that check with `hasScope('record.archive')`. @jpye-finch would this be enough or do we still need to some way check if the status is validated and then based on that and some scopes do different actions?